src/utils/floating-ui.ts

```diff
@@ -27,29 +27,44 @@ async function patchFloatingUiForNonChromiumBrowsers(): Promise<void> {
     platform: string;
   }
 
+  function getUAData(): NavigatorUAData | undefined {
+    return (navigator as any).userAgentData;
```

Comment:
I assume you had to assert `navigator` as `any` because `userAgentData` is experimental and not included in its type yet?

https://developer.mozilla.org/en-US/docs/Web/API/Navigator/userAgentData#browser_compatibility

Reply:
Correct and this particular code is based on `@floating-ui/dom` https://github.com/floating-ui/floating-ui/blob/f63411e85ae12bfe0c983e7c8104c57eb7849d0a/packages/dom/src/utils/userAgent.ts.